client/src/views/TrackViewer/Viewer.vue

```diff
@@ -176,6 +179,20 @@ export default defineComponent({
       }
     }
 
+    //Selection of next track while seeking to position
+    function handleSelectNext(delta: number) {
+      const newTrack = selectNextTrack(delta);
+      if (newTrack !== null) {
+        selectTrack(newTrack, false);
+        if (selectedTrackId.value !== null) {
```

Comment:
this condition is redundant.  `selectTrack()` is a synchronous function that sets `selectedTrackId`, so if `newTrack !== null` then this cannot ever evaluate false.

Reply:
Oops, my mistake, I should have the getTrack  utilize the NewTrack instead of the selectedTrackId.  It was having an issue with the linter because it didn't know selectedTrackId.  Makes more sense to just use the NewTrack because we have the Id already.